src/current/_includes/molt/molt-docker.md

```diff
@@ -0,0 +1,78 @@
+### Performance
+
+MOLT Fetch and Verify are likely to run more slowly in a Docker container than on a local machine. To improve performance, increase the memory or compute resources on your Docker container.
+
+{% if page.name == "molt-fetch.md" %}
+### Authentication
+
+{{site.data.alerts.callout_info}}
```

Comment:
I am not sure that this info needs to be in a Note callout. It seems to be more like an intro to the section.

Reply:
Removed the callout formatting.